pkg/secrets/secrets.go

```diff
@@ -52,6 +52,22 @@ func GenerateUserData(publicKey ssh.PublicKey) (*core.Secret, error) {
 		},
 		Data: map[string][]byte{
 			"userData": []byte(`<powershell>
+			function Get-RandomPassword {
+				Add-Type -AssemblyName 'System.Web'
+				return [System.Web.Security.Membership]::GeneratePassword(16, 2)
+			}
+
+			# Check if the capi user exists, this will be the case on Azure, and will be used instead of Administrator
```

Comment:
As an aside I wonder if we can do this on Azure also? It is also making me wonder if we should just create a new `core` user instead like we wanted to in the past.

Reply:
We can re-open https://issues.redhat.com/browse/WINC-430
This could cause issues with people using custom images

Reply:
>This could cause issues with people using custom images

Why would creating a core user cause issues?

Reply:
I'm thinking of a case where the Administrator user has been set up with certain permissions in the image, or theres security software installed special casing the Administrator user. Its nothing we cant doc around.